build.cake

```diff
@@ -233,6 +235,32 @@ Task("Create-Tools-Package")
             }
         });
     });
+
+Task("Create-Installer")
+    .IsDependentOn("Copy-Files")
+    .Does(() => {
+        CopyDirectory(binDir, installerDir + Directory("lib") + Directory("net45"));
+        CopyFileToDirectory("./src/Wyam.nuspec", installerDir);
+        var nuspec = GetFiles(installerDir + File("Wyam.nuspec")).FirstOrDefault();
+
+        NuGetPack(nuspec, new NuGetPackSettings
```

Comment:
Ah, got it. That makes sense. For a "tools" package (I don't know if that's the official name for these or not), you put everything under /tools. The NuGet clients know that if there's stuff in /tools that it should do a little magic and make it available for use in the Package Manager Console.

I've never actually used Squirrel - so found this a bit interesting when going through their docs just now:

> Squirrel is expecting a single lib / net45 directory provided regardless of whether your app is a net45 application

I may suggest/submit a PR to allow that to be customized or to use /tools as a fallback. In the meantime let's just leave this as-is since it works.